tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json.schemastore.org/tsconfig",
+    "compilerOptions": {
+        "module": "NodeNext",
+        "target": "esnext",
+        "moduleResolution": "nodenext",
+        "allowJs": true,
+        "declaration": true,
+        "emitDeclarationOnly": true,
+        "declarationDir": "types",
+        "forceConsistentCasingInFileNames": true,
+        "checkJs": false,
+        "strict": true
+    },
+    "include": [
+        "esm/**/*.js"
+    ]
+}
```

Comment:
please add a new line for GitHub sake as that missing new line thing is bothering the robots 😅